### PR TITLE
7381_Minullin_6_4

### PR DESCRIPTION
**Pull request by Fikafusik**
Строил на основе примера, который дан был материале практики: https://keras.io/examples/mnist_cnn/
Dropout исключил, поскольку переобучения не наблюдалось.
Перемешал, чтобы узнать, как перемешивать несколько массивов одной длины с сохранением отношения между массивами (ну, не только). 
Точность модели на данных для обучения 100%, на тестовых 98%. В ПР графики потерь и точности. 
Метки Square и Circle заменил на 0 и 1

### CODE REVIEW

**Comment by Ksenox** [comment]
Зачем метки заменять? Можно же использовать LabelEncoder

**Comment by Fikafusik** [comment]
Так быстрее.

**Comment by Ksenox** [comment]
Сделайте так, чтобы использовался LabelEncoder

**Comment by Ksenox** [comment]
2 Балла
